src/pages/Authentication/pages/ForgotPassword/ForgotPassword.tsx

```diff
@@ -0,0 +1,131 @@
+import axios, { AxiosResponse, AxiosError } from "axios";
+import { useState, ChangeEvent, FocusEvent, FormEvent } from "react";
+import { Link } from "react-router-dom";
+import { Button } from "../../../../components";
+import Input from "../../../../components/Input";
+import AuthenticationForm from "../../components/AuthenticationForm";
+import NotificationCard from "../../components/NotificationCard";
+import OuterDiv from "../../components/OuterDiv";
+import { baseUrl, validate } from "../../utils/helpers";
+import BackgroundWrapper from "@/Wrappers/BackgroundWrapper";
+import { ApiResponse } from "@/utils/interfaces";
+
+interface EmailValue {
+  email?: string;
+}
+
+interface ErrorState {
+  email?: boolean;
+  status?: boolean;
+}
+
+
+
+const ForgotPassword = () => {
+  const [emailValue, setEmailValue] = useState<EmailValue>({});
```

Comment:
Isnt email value a string
Why initialize it as an object
I assume we czn do something like useState<string>("")